### PR TITLE
Update 08-motivation.md

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR updates the lesson on motivation, including: 

1) Updating key points, objectives and questions to keep to a limited number and reflect current episode content.

2) Changing "students" to "learners" throughout.

3) Removing some text that was previously moved to the episode on expertise. 

4) Reorganizing section on demotivation to split into two sections - one on things we say at workshops that can be demotivating and the other on larger psychological and societal demotivators. 

5) Adding challenges about accessibility and imposter syndrome.

6) Updating time estimates.

7) Removing outdated links.

### CODE REVIEW

File: _episodes/08-motivation.md [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
-- "Be careful not to remind learners of negative stereotypes when teaching."
-- "We're all faking it."
-- "Don't teach or learn alone."
-- "Belief that competence comes with practice improves learning outcomes."
-- "Measures taken to improve accessibility aid everyone."
-- "Measures taken to make learning more inclusive aid everyone."
+- "A positive learning environment helps people concentrate on learning."
+- "People learn best when they see the utility in what they're learning, so teach what's most immediately useful first."
+- "Imposter syndrome is a powerful force, but can be overcome."
+- "Accessibility benefits everyone."
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: Just reading these four keypoints makes me happy. :) This stuff is so important and I'm so glad we teach it!

File: _episodes/08-motivation.md [hunk 2]
```diff
@@ -1,43 +1,41 @@
...
 need encouragement.  This section discusses typical ways that
-learners are motivated (and can be demotivated!) and describes ways that communities
-of practice can be welcoming (or threatening) to new members.
+learners are motivated (and can be demotivated!) and provides practice opportunities
+for you to become confident in motivating your learners.
 
 ## Creating A Positive Learning Environment
 
-Creating a positive learning environment is a crucial first step to setting the stage for student success. As instructors, it is crucial to establish the workshop setting as a safe space for learning. Establishing a safe space for learning is a combination of many factors:
+Creating a positive learning environment is a crucial first step to setting the stage
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: crucial -> important
Just to not repeat in the next line.

File: _episodes/08-motivation.md [hunk 3]
```diff
@@ -222,8 +193,8 @@ to alienate a classroom and cause learners to tune out.
 *   Pretend to know more than you do.  People will actually trust you
     more if you are frank about the limitations of your knowledge, and
     will be more likely to ask questions and seek help.
-*   Use the J word ("just");
-    this signals to the learner that
+*   Use the J word ("just") or other demotivating words we talked about in a 
+    previous lesson. These signal to the learner that
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: link to previous lesson?

File: _episodes/08-motivation.md [hunk 4]
```diff
@@ -233,93 +204,13 @@ to alienate a classroom and cause learners to tune out.
     are in the wrong place, and it may prevent them from asking questions in
     the future. (This idea is due to the Recurse Center's [Social Rules][recurse-social-rules]).
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: Is it worth calling out contempt/scorn explicitly in this list?  It's described in the following paragraph, so I think the content is there, but I do think of it as its own category of "don't-do-this"

File: _episodes/08-motivation.md [hunk 5]
```diff
@@ -330,6 +221,35 @@ it with more positive and motivating word choices.
...
 {: .challenge}
 
+## Psychological Demotivators
+
+As instructors, we can learn to avoid talking disparagingly about our learners' choice
+of text editors and levels of technical knowledge. This helps a lot in avoid 
+demotivating learners. There are other factors, however, that contribute to demotivation, 
+some of which are either systemic, or built into our psychological makeup as human beings. 
+We can still have a positive impact in fighting these demotivators by thinking carefully about 
+the language that we use and how we interact with our learners.
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: Great transition / set up paragraph.

File: _episodes/08-motivation.md [hunk 6]
```diff
@@ -344,22 +264,30 @@ it with more positive and motivating word choices.
...
+>
+> Software Carpentry and Data Carpentry share a
+> [code of conduct]({{ site.swc_site }}/conduct/),
+> and all participants in our workshops are required to abide by it.
+> Its details are important,
+> but the most important thing about it is that it exists:
+> knowing that we have rules
+> tells people a great deal about our values
+> and about what kind of learning experience they can expect.
+{: .callout}
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: It also puts everyone on an even playing field (relevant to this particular topic of dealing with stereotypes)

File: _episodes/08-motivation.md [hunk 7]
```diff
@@ -433,14 +371,12 @@ while Liz Henry's blog post about
 [accessibility at conferences][conference-accessibility]
 has a good checklist for people with mobility issues,
 and [this interview][deaf-accessibility]
-with Chad Taylor is a good introduction to issues faced by the hearing impaired.
+with Chad Taylor is a good introduction to issues faced by those with no or limited hearing.
```
Comment (ChristinaLK) [inline _episodes/08-motivation.md]: Add resources for color blindness?

File: _episodes/08-motivation.md (hunk 4 above)
Comment (ErinBecker) [inline _episodes/08-motivation.md]: Added an attempt at this.

File: _episodes/08-motivation.md (hunk 7 above)
Comment (ErinBecker) [inline _episodes/08-motivation.md]: Added new issue for this (https://github.com/ErinBecker/instructor-training-reorg/issues/170).

**Comment by ErinBecker** [comment]
Merging based on approval from @ChristinaLK.

**Pull request merged**